### PR TITLE
data flows

### PR DESCRIPTION
**Pull request by jhadobe**
This PR adds data flow documentation to the existing PR #184

### CODE REVIEW

File: src/pages/app-development/starter-kit/customers.md [hunk 1]
```diff
@@ -10,4 +10,39 @@ keywords:
  - Tools
 ---
 
-# Customers data flow
+# `customer` data flow
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
```
Comment (danperezAdobe) [inline src/pages/app-development/starter-kit/customers.md:16]: Should we include a diagram similar like this one in each one of the data flows? https://wiki.corp.adobe.com/display/EntComm/Product+Data+Object

**Review by danperezAdobe — APPROVED** [review]

File: src/pages/app-development/starter-kit/customers.md [hunk 2]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+## Create
+
+Adobe Commerce origination
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:19]: Global comment: Can we make this a full sentence?

Adobe Commerce supports the following ways to create a customer.

(I'm not happy wiith that suggestion, but I'm trying to start the sentence with AC and not use passive voice

File: src/pages/app-development/starter-kit/customers.md [hunk 3]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+
+- A new customer registers in Adobe Commerce (sign up or request)
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:21]:
```suggestion
- A shopper registers on the Create an Account page
```

File: src/pages/app-development/starter-kit/customers.md [hunk 4]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+- A new customer registers in Adobe Commerce during or after purchase (sign up and purchase)
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:22]:
```suggestion
- A shopper registers during or after purchase (sign up and purchase)
```

File: src/pages/app-development/starter-kit/customers.md [hunk 5]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+- An admin user registers a new customer
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:23]: Customers can also be created with REST and GraphQL. I think GraphQL would only be used to complete the UI actions, but REST might be used to create accounts in bulk

File: src/pages/app-development/starter-kit/customers.md [hunk 6]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+
+Back office integration origination
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:25]: Make this a sentence too

File: src/pages/app-development/starter-kit/customers.md [hunk 7]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+
+- A new or existing customer engages the company through sales, a partner, or a CRM, when records and emails are synched into Adobe Commerce.
+
+## Update
+
+Adobe Commerce origination
+
+- The customer edits information via My Account
+- An admin User modifies existing customer information (including assignment to a different customer group)
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:34]:
```suggestion
- An admin user or process modifies existing customer information (including assignment to a different customer group)
```

By process I mean REST/GraphQL

File: src/pages/app-development/starter-kit/orders.md [hunk 8]
```diff
@@ -10,4 +10,23 @@ keywords:
...
-# Orders data flow
+# `order` data flow
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
+## Create
+
+Adobe Commerce origination
+
+- A purchase is made in Adobe Commerce
```
Comment (keharper) [inline src/pages/app-development/starter-kit/orders.md:21]:
```suggestion
- A customer or guest makes a purchase
```

File: src/pages/app-development/starter-kit/products.md [hunk 9]
```diff
@@ -10,4 +10,33 @@ keywords:
...
+- A new product is created in the back office system
+- A bulk import from an existing Product Information Management (PIM) system creates new products
+
+## Update
+
+Adobe Commerce origination
+
+- An Adobe Commerce API updates products from external systems
+- An admin user modifies the existing product
+- An admin user modifies the type of product (for example, simple > configurable, simple > digital)
```
Comment (keharper) [inline src/pages/app-development/starter-kit/products.md:37]:
```suggestion
- An admin user modifies the type of product (for example, from simple to configurable or virtual)
```

File: src/pages/app-development/starter-kit/shipments.md [hunk 10]
```diff
@@ -10,4 +10,13 @@ keywords:
  - Tools
 ---
 
-# Shipments data flow
+# `shipment` data flow
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
+## Create
+
```
Comment (keharper) [inline src/pages/app-development/starter-kit/shipments.md:18]: I think the Commerce and backoffice integration bullets are the same

File: src/pages/app-development/starter-kit/stock.md [hunk 11]
```diff
@@ -10,4 +10,42 @@ keywords:
...
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
+The following sections describe when events are fired for this object.
+
+## Create (stock hierarchy)
+
+Adobe Commerce origination
+
+- A new Stock Source is created in the Admin panel
```
Comment (keharper) [inline src/pages/app-development/starter-kit/stock.md:23]: or with a REST API

No need to capitalize Stock or Source

File: src/pages/app-development/starter-kit/stock.md [hunk 12]
```diff
@@ -10,4 +10,42 @@ keywords:
...
+
+Back office integration origination
+
+- A new Stock source is created in the backoffice system
+
+## Update (stock hierarchy)
+
+Adobe Commerce origination
+
+- Admin User modifies Stock source
```
Comment (keharper) [inline src/pages/app-development/starter-kit/stock.md:33]:
```suggestion
- Admin user modifies a stock source
```

File: src/pages/app-development/starter-kit/customers.md [hunk 13]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+Adobe Commerce supports the following update methods:
+
+- The customer edits information via My Account
+- An admin user or process modifies existing customer information (including assignment to a different customer group)
+
+Backoffice integrations support the following update methods:
+
+- An existing customer engages the company through sales, a partner, or a CRM (including assignment to a different customer group)
+
+## Create/Update
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:40]:
```suggestion
## Customer group create/update
```

File: src/pages/app-development/starter-kit/customers.md [hunk 14]
```diff
@@ -10,4 +10,39 @@ keywords:
  - Tools
 ---
 
-# Customers data flow
+# `customer` data flow
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
+## Create
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:17]:
```suggestion
## Customer create
```

File: src/pages/app-development/starter-kit/customers.md [hunk 15]
```diff
@@ -10,4 +10,39 @@ keywords:
...
+
+- A shopper registers on the Create an Account page
+- A shopper registers during or after purchase (sign up and purchase)
+- An admin or an API creates a new customer
+
+Backoffice integrations support the following create methods:
+
+- A new or existing customer engages the company through sales, a partner, or a CRM, when records and emails are synched into Adobe Commerce.
+
+## Update
```
Comment (keharper) [inline src/pages/app-development/starter-kit/customers.md:29]:
```suggestion
## Customer update
```

**Review by keharper — APPROVED** [review]

**Pull request merged**